features/support/env.rb

```diff
@@ -6,7 +6,7 @@
 
 ENV['RAILS_ENV'] = 'test'
 
-require 'simplecov'
+require 'simplecov' if ENV["COVERAGE"] == "true"
```

Comment:
I think it is OK to delete or squash branch commits instead of reverting them.

Reply:
Yes, absolutely! But I prefer to add separate commits initially to keep track of the changes I make. Once I have a final working version, I cleanup history.

In this case I realized that the removal of optional coverage is not good, because when you run single specs, so get an annoying delay for coverage report generation. So I restored that and wanted to check that I'm not breaking anything. After that I'll cleanup the branch.